pom.xml

```diff
@@ -730,7 +730,7 @@
                         <instructions>
                             <!-- Property bundle.activator.class is defined in individual plugins -->
                             <Bundle-Activator>${bundle.activator.class}</Bundle-Activator>
-                            <Export-Package/>
+                            <Export-Package />
```

Comment:
why this change?

Reply:
It was added by IntelliJ. Not sure why. But let me take it out